packages/auth/src/providers/cognito/apis/signUp.ts

```diff
@@ -31,56 +35,68 @@ import { signUpClient } from '../utils/clients/SignUpClient';
 export async function signUp(
 	signUpRequest: SignUpRequest<CognitoUserAttributeKey, CognitoSignUpOptions>
 ): Promise<AuthSignUpResult<AuthStandardAttributeKey | CustomAttribute>> {
+    const username = signUpRequest.username
+	const password = signUpRequest.password
```

Comment:
Seems like this can just be destructured as well
```suggestion
    const { username, password } = signUpRequest
```